css/css-values/percentage-rem-low-ref.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
```

Comment:
Can you move this file into the references/ subdirectory?

Reply:
I should probably just delete it?